addons/l10n_br_pix/views/res_bank_views.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="view_partner_bank_form_inherit_account" model="ir.ui.view">
+        <field name="name">res.partner.bank.form.inherit</field>
+        <field name="model">res.partner.bank</field>
+        <field name="inherit_id" ref="base.view_partner_bank_form"/>
+        <field name="arch" type="xml">
+            <field name="include_reference" position="after">
+                <p invisible="country_code != 'BR'">
+                    <a href="https://www.odoo.com/documentation/17.0/applications/finance/fiscal_localizations/brazil.html" target='_blank'>Documentation</a>
```

Comment:
Do we have a doc PR?

Reply:
We'll put up a draft PR today. If it's ok for you we would like to not wait on that, this feature is fairly straightforward anyway.

Reply:
https://github.com/odoo/documentation/pull/8067